test/run/idl.mo

```diff
@@ -1,33 +1,35 @@
 //MOC-ENV MOC_UNLOCK_PRIM=yesplease
 import Prim "mo:⛔";
 
-func serUnit() : Blob = (prim "serialize" : () -> Blob) ();
-func deserUnit(x : Blob) : () = (prim "deserialize" : Blob -> ()) x;
+func serUnit() : Blob = to_candid ();
```

Comment:
Hmm, its a bit weird that this file is testing both the new syntax and the old prims. Maybe revert this one to what it was, and write an additional one, derived from it, that uses the new syntax throughout (even for the array examples at the end that you kept in the old format).

Reply:
Okay.

It's not weird if you think (like I did until now) that the new surface syntax was replacing these undocumented primitives.  Why even expose them now?  Is there some benefit to having both approaches?

In any case, if we have both, it makes sense to test them both, and to separate the tests.  Thanks.